src/medias/images/UGT_Asset_FlagSelector_EU.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+export interface ImgFlagEUProps extends React.SVGProps<SVGSVGElement> {
+  alt: string;
+}
+
+export const ImgFlagEU: React.FunctionComponent<ImgFlagEUProps> = ({ alt, ...props }) => {
```

Comment:
*non blocking*
```suggestion
export const ImgFlagEu: React.FunctionComponent<ImgFlagEuProps> = ({ alt, ...props }) => {
```
would be more consistent with the others

Reply:
hm. did it on purpose this ways since EU is an abbreviation not a shortening of the word